oak-run/src/main/java/org/apache/jackrabbit/oak/run/DataStoreCommand.java

```diff
@@ -212,12 +211,11 @@ private void execute(NodeStoreFixture fixture, DataStoreOptions dataStoreOpts, O
 
                                 while (idIter.hasNext()) {
                                     String id = idIter.next();
-                                    final Joiner delimJoiner = Joiner.on(DELIM).skipNulls();
                                     // If --verbose is present, convert blob ID to a backend friendly format and
                                     // concat the path that has the ref. Otherwise simply add the ID to the o/p file
                                     // as it is.
                                     String line = dataStoreOpts.isVerbose() ?
-                                        VerboseIdLogger.encodeId(delimJoiner.join(id, escapeLineBreak(nodeId)),
+                                        VerboseIdLogger.encodeId(String.join(DELIM, id, escapeLineBreak(nodeId)),
```

Comment:
Isn't it theoretically possible that nodeId is null?

Reply:
it might